src/components/companyList.js

```diff
@@ -68,18 +69,18 @@ function Companies({ sector }) {
         <div>Finance APP . LIST </div>
         <div><FontAwesomeIcon icon={faGear} /></div>
       </header>
-      <div className={style.companiesHeaderContainer}>
+      <div className={companiesHeaderContainer}>
         <FontAwesomeIcon icon={ICONS[sector]} />
         <div>{sector}</div>
       </div>
 
-      <section className={style.companiesSearch}>
+      <section className={companiesSearch}>
         <FontAwesomeIcon icon={faMagnifyingGlass} />
         <input ref={filterRef} type="text" placeholder="search" onKeyDown={setFilterField} />
         <button type="button" onClick={setFilterFieldOnClick}>search</button>
       </section>
 
-      <section className={style.companiesContainer}>
+      <section className={companiesContainer}>
         {
           companies
             .filter((company) => company.companyName.toLowerCase().includes(filter.toLowerCase())
```

Comment:
- [ ] As the previous reviewer suggested that your icon should be in the center as you can see in the required design :+1: to make your project look attractive and professional :+1: Now your icon is going out of your screen on the left side :thinking:




Your Implementation | Figma Design
-- | --
![image](https://user-images.githubusercontent.com/93039359/232988239-a3381a01-5c77-4f12-baa2-df1526d49039.png)  | ![image](https://user-images.githubusercontent.com/93039359/232988904-a5327469-1ccc-4c22-8217-cbaa98fce966.png)

Reply:
I have done changes